app/models/restaurant.rb

```diff
@@ -1,5 +1,5 @@
 class Restaurant < ActiveRecord::Base
-  has_many :reviews, :dependent => :destroy
+  has_many :reviews
```

Comment:
is a review relevant when a restaurant is destroyed? I think the destroy dependency is appropriate here.